tests/utils/dev-server.js

```diff
@@ -43,6 +43,7 @@ const startServer = async ({ cwd, env = {}, args = [] }) => {
               }
             })
             ps.kill()
+            // eslint-disable-next-line no-shadow
             await Promise.race([ps.catch(() => {}), new Promise((resolve) => setTimeout(resolve, 1000))])
```

Comment:
This conflicts with a ESLint rule which requires promise first callback to be named `resolve`.
The real solution here is not to nest two `Promise`, but that'd be a bigger refactoring.